contents.json

```diff
@@ -512,6 +512,18 @@
       "city": "Boston",
       "callforpaper": true,
       "emojiflag": "🇺🇸"
+    },
+    {
+      "title": "ADDC",
+      "year": 2018,
+      "startdate": "2018/07/04",
+      "enddate": "2018/07/06",
+      "where": "Avinguda Diagonal, 547, 08029 Barcelona, Spain",
+      "homepage": "http://www.addconf.com",

```

Comment:
@pepibumur can you put https instead? validator fails otherwise because it makes a strict check on links (I know sometimes is too strict)

Reply:
I just saw the branch was merged? Should I create another one with the fix or will you push it to master?

Reply:
I already fixed and push to master